admin/worker_v2.m.template

```diff
@@ -13,8 +13,36 @@ function [ok,err_mess,je]=worker_v2(worker_controls_string)
 %              contains only minimal initialization information, namely the
 %              folder name where the job initialization data are located on
 %              a remote system.
+je = [];
 if isempty(which('herbert_init.m'))
     herbert_on();
+    %horace_on('c:\Users\abuts\Documents\developing_soft\Horace_3_4\horace_core');
 end
-DO_LOGGING = true; % if true, write extensive log information.
-[ok, err_mess,je] = parallel_worker(worker_controls_string,DO_LOGGING);
+DO_LOGGING = false;
+try
+    [ok, err_mess,je] = parallel_worker(worker_controls_string,DO_LOGGING);
+catch ME1 % intercepted exception in processing failure or some odd bug indeed
+    ok = false;
+    if exist('je','var') && isa(je,'JobExecutor') %exception thrown but JE is defined
+        if isnumeric(je.labIndex) && ~isempty(je.labIndex)
+            log_file = fullfile(tempdir,sprintf('WORKER_V2_Node#%d_failure.log',je.labIndex));
+        else
+            log_file = fullfile(tempdir,sprintf('WORKER_V2_Node_JE_common_failure.log'));
```

Comment:
Why not use a process ID here (`feature('getpid')`)? Then, if we get multiple failures, the processes are not all trying to write to the same file

Reply:
good point -- not sure if it works but worth trying